src/util.rs

```diff
@@ -44,6 +44,25 @@ pub fn history() -> web_sys::History {
     window().history().expect("Can't find history")
 }
 
+/// Convenience function to access the `web_sys::HtmlDocument`.
+pub fn html_document() -> web_sys::HtmlDocument {
+    wasm_bindgen::JsValue::from(document()).unchecked_into::<web_sys::HtmlDocument>()
+}
+
+/// Convenience function to get all cookies from the current `HtmlDocument`
+#[cfg(feature = "cookies")]
+pub fn cookies() -> cookie::CookieJar {
```

Comment:
Can `html_document().cookie()` and `::parse_encoded(..)` fail in a real-world scenario? => Is it ok that we ignore errors?

Reply:
> Can `html_document().cookie()` and `::parse_encoded(..)` fail in a real-world scenario?

I have no idea :-\

> => Is it ok that we ignore errors?

We could at least return an `Option<CookieJar>`

Reply:
> We could at least return an Option<CookieJar>

Yeah, users should at least have to add `uwrap` and think about it. Or alternative is `Result<CookieJar, ()> so we can add more details in the error in the future without "big" breaking change.

Reply:
I changed it to `Option` for now.

The be more precise we need `Result<Option<CookieJar>, ()>` or `Result<Vec<Cookie>, ()>` because I don't want to return a `CookieJar` that is empty.